tasks/targets.py

```diff
@@ -476,3 +476,15 @@ def exists(self):
                     function_name=self._function_name)
 
         return len(self._session.execute(query).fetchall()) > 0
+
+
+class URLTarget(Target):
+    def __init__(self, url):
+        self.value = url
+        self.path = url
+
+    def exists(self):
+        return True
```

Comment:
Related to https://github.com/CartoDB/bigmetadata/pull/621/files#diff-f1267b51a30fc6c52919dc99cf97448dR1948